web/tx_attempts_controller.go

```diff
@@ -32,3 +35,19 @@ func (tac *TxAttemptsController) Index(c *gin.Context) {
 		c.Data(200, MediaType, buffer)
 	}
 }
+
+// Show returns the details of a Transasction Attempt.
+// Example:
+//  "<application>/txattempts/:TxHash"
+func (tac *TxAttemptsController) Show(c *gin.Context) {
+	hash := common.HexToHash(c.Param("TxHash"))
+	if tx, err := tac.App.GetStore().FindTxByAttempt(hash); err == orm.ErrorNotFound {
+		c.AbortWithError(404, errors.New("Transaction Attempt not found"))
```

Comment:
The error message is a little off, since you are actually looking for transactions. It should be

```
Transaction not found
```

Reply:
Fixed in the new controller.